.github/actions/javascript/validateReassureOutput/validateReassureOutput.js

```diff
@@ -1,7 +1,8 @@
 const core = require('@actions/core');
+const fs = require('fs');
 
 const run = () => {
-    const regressionOutput = JSON.parse(core.getInput('REGRESSION_OUTPUT', {required: true}));
+    const regressionOutput = JSON.parse(fs.readFileSync('.reassure/output.json', 'utf8'));
```

Comment:
Where do you save the output to this file?

Reply:
Output is automatically saved in `.reassure/output` after reassure tests finish

Reply:
@mountiny I tested only reassure tests so far. How we can test GitHub actions locally? Is It possible?

Reply:
Ah fair point, i dont think that is possible yet